pkg/repository/udmrepo/repo_options.go

```diff
@@ -66,13 +65,12 @@ const (
 	StoreOptionGenReadOnly        = "readOnly"
 
 	StoreOptionCacheLimit = "cacheLimitMB"
-	StoreOptionCacheDir   = "cacheDir"
 
-	ThrottleOptionReadOps       = "readOPS"
-	ThrottleOptionWriteOps      = "writeOPS"
-	ThrottleOptionListOps       = "listOPS"
-	ThrottleOptionUploadBytes   = "uploadBytes"
-	ThrottleOptionDownloadBytes = "downloadBytes"
+	ThrottleOptionReadOps       = "ThrottleReadOPS"
```

Comment:
Sorry for misleading of my previous comment. We add `throttle` for each of the options, but as the convention, we need to make the first word lowercase, like:
```
    ThrottleOptionReadOps       = "throttleReadOPS"
	ThrottleOptionWriteOps      = "throttleWriteOPS"
	ThrottleOptionListOps       = "throttleListOPS"
	ThrottleOptionUploadBytes   = "throttleUploadBytes"
	ThrottleOptionDownloadBytes = "throttleDownloadBytes"
```